engine/transaction/draft/create_draft_test.go

```diff
@@ -0,0 +1,39 @@
+package draft_test
```

Comment:
As I recall - the package names should not contain "underscore"

Reply:
![image](https://github.com/user-attachments/assets/184b2559-bb69-4406-b8dd-9d395bbb68b2)
https://pkg.go.dev/testing

The `_test` package gives you a possibility to test your solution from the perspective of the client - component from different package, because it is treated as different package, but allowed to be in the same package as the other in that directory.

Reply:
Of course - It was indeed late hour of work that I haven't notice that 🙈